src/main/java/ws/wamp/jawampa/WampClient.java

```diff
@@ -842,6 +853,12 @@ public void run() {
                 
                 final long requestId = IdGenerator.newLinearId(lastRequestId, requestMap);
                 lastRequestId = requestId;
+
+                ObjectNode options = null;
+                if (flags == PublishFlags.DontExcludeMe) {
+                    options = new ObjectMapper().createObjectNode();
```

Comment:
Should use the objectMapper field of the client class instead of creating a new one